examples/operator/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "operator.fullname" . }}-controller-manager
+  labels:
+  {{- include "operator.labels" . | nindent 4 }}
+  annotations:
+    k8s.acme.org/some-meta-data: ACME Inc.
```

Comment:
should `k8s.acme.org/some-meta-data: ACME Inc.` be moved to `values.yaml`?

Reply:
@arttor does helmify do that with labels?